ansible/values.yml

```diff
@@ -58,7 +58,7 @@ fabric_channels:
 
 ######## Templated variables with default values, DO NOT adjust unless in advanced mode ###########
 cloud_provider: aws
-console_image: ghcr.io/senofi/fabric-console
+console_image: ghcr.io/openidl-org/fabric-console
```

Comment:
@TsvetanG we may need to leave this in the Senofi repo for the time being

Reply:
yes for now I agree.  We need to have a chat on that with Sean. We want to contribute the vault integration to the fabric console project in the future.